07-reputation/01-configuration.md

```diff
@@ -0,0 +1,8 @@
+## Configuration
+
+Nodes need to know the values of the variable parameters of the algorithm for calculating reputation values using the EigenTrust algorithm. These parameters are constant, but in general they can be changed in order to adapt the operation of the algorithm. For synchronization and correct operation of all nodes, the values are moved to the `Netmap` contract and are read from it at the beginning of each epoch.
```

Comment:
1 and 3 are fixed. 2 is questionable: i wanted to say the following: "usually they are constant but it is possible to change them theoretically(in fact it will not be done)". literally translated from russian, it sounds like "generally speaking". is "normally" the correct word in such cases?

Reply:
I think it can be omitted then: "These parameters are constant but can be changed if one needs to adapt the algorithm." "Generally speaking" and "in general" doesn't sound natural here.